src/model/styles.rs

```diff
@@ -433,7 +433,7 @@ impl<'a> StyleChain<'a> {
             values
                 .next()
                 .map(|value| value.fold(next(values, styles, default)))
-                .unwrap_or_else(|| default())
+                .unwrap_or(default())
```

Comment:
This could be expensive

Reply:
There's also a `unwrap_or_default()` method for this exact use case.

Reply:
That doesn't work here because `default` is a closure provided to this function, not `Default::default`.